src/main/java/com/imaee/propinq/neighborhood/controllers/implementations/LocalityController.java

```diff
@@ -0,0 +1,43 @@
+package com.imaee.propinq.neighborhood.controllers.implementations;
+
+import com.imaee.propinq.neighborhood.controllers.interfaces.ILocalityController;
+import com.imaee.propinq.neighborhood.controllers.requests.LocalityRequest;
+import com.imaee.propinq.neighborhood.controllers.responses.LocalityResponse;
+import com.imaee.propinq.neighborhood.services.interfaces.ILocalityService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+public class LocalityController implements ILocalityController {
+    @Autowired
+    private ILocalityService localityService;
```

Comment:
Para evitar poner el `@Autowired` en cada uno de los inyectables que necesites, podés usar Lombok para que te genere el constructor. Spring es capaz de reconocer qué inyectables necesita una de tus clases a partir de los constructores. Magia

```suggestion
@RestController
@AllArgsConstructor
public class LocalityController implements ILocalityController {

    private final ILocalityService localityService;
```